crates/c-api/src/module.rs

```diff
@@ -2,11 +2,12 @@ use crate::{handle_result, wasmtime_error_t};
 use crate::{wasm_byte_vec_t, wasm_exporttype_vec_t, wasm_importtype_vec_t};
 use crate::{wasm_exporttype_t, wasm_importtype_t, wasm_store_t};
 use std::ptr;
-use wasmtime::{HostRef, Module};
+use wasmtime::{HostRef, Module, Store};
 
 #[repr(C)]
 #[derive(Clone)]
 pub struct wasm_module_t {
+    pub(crate) store: HostRef<Store>,
```

Comment:
Theoretically, yes, we can remove. It will drive wasmtime_instance_new signature change too, and renders wasm_module_share / wasm_module_obtain useless. I would like to defer this change for future work.

Reply:
I think changing signatures is fine, but if you'd prefer to defer that's ok too. Can you be sure an issue is filed about this?